### ISSUE TITLE
[Feature] native mermaid support for md diagrams

### ISSUE THREAD

**Issue by aleksAperans** [fern-api/fern#2857]
### Problem description

Looking for a way to better handle visualizations and diagrams via markdown in fern

### Why would it be useful?

leveraging mermaid md would enable our team to maintain a single source of truth for all of our visuals. We're not blocked by this, so its very much a nice to have but if there is enough interest across the fern community, I suspect it would streamline documentation activities.

### Describe the solution (optional)

A concise description of what the feature should be.

### Additional context

https://mermaid.js.org/

**Comment by dannysheridan** [comment]
Closing as uploading an image works for now! Can revisit if you or others need this down the line.
